pkg/agent/cniserver/interface_configuration_linux.go

```diff
@@ -228,7 +239,75 @@ func (ic *ifConfigurator) configureContainerSriovLink(
 	hostIface.Name = vfIFName
 	klog.V(2).Infof("hostIface.Name: %s, hostIface.Mac: %s, vfIFName: %s", hostIface.Name, hostIface.Mac, vfIFName)
 
-	return ic.moveVFtoContainerNS(vfIFName, containerID, containerNetNS, containerIfaceName, mtu, result)
+	newVFIFName := getVFInterfaceNameWithSuffix(vfIFName)
+	err = ic.netlink.LinkSetAlias(link, newVFIFName)
+	if err != nil {
+		return fmt.Errorf("failed to set alias %s for VF netdevice %s: %v", newVFIFName, vfIFName, err)
+	}
+	klog.InfoS("link's alias has been updated", "alias", newVFIFName)
```

Comment:
I feel like it should be `V(2)`
The log message should also be capitalized

Same comments for other new instances of `klog.InfoS` below